src/shared/common/components/workstack-tag.jsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const { fetchTagType } = require('../../../../server/config/tagType/tagType');
+
 export default function Tags(props) {
     let tags = [];
+    if (props.tags) {
+        tags = props.tags.map(tag => {
+            let tagDetails = fetchTagType(tag);
 
-    if (props.row.tag && props.row.tag.length > 0) {
-        tags = props.row.tag.map(tag => {
-            return <span key={tag} className="govuk-tag govuk-!-margin-right-1">{tag}</span>;
+            return <span key={tag}
+                className={tagDetails?.displayClass ?? 'govuk-tag govuk-!-margin-right-1'}>{tagDetails?.label ?? tag}</span>;
         });
     }
 
     return tags;
 }
 
 Tags.propTypes = {
-    row: PropTypes.object,
     tag: PropTypes.array
```

Comment:
We could ensure a default prop here and further get rid of the tag exists check? 

Look up React defaultProps...